### PR TITLE
Fixes #31 update documentation of hasUniqueValues

### LINKED ISSUE #31: 'hasUniqueValues'
Documentation states 

`#' Remove duplicate values from data`

Either the docu is wrong (which I hope), or the function should be renamed.

Also the argument `data` does not make it clear that it requires a data frame. Wait, must it be a data frame? If not - wrong docu.

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
@msevestre The same issue as [here](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/commit/9f1c4f4b99630ff84a48fa38d687aa27d91d8661#commitcomment-65806469) is observed in this PR.

This is good. Because based on what is common to both of your machines, we can figure out what is causing these massive changes.

Are you by any chance using `VS Code` instead of `RStudio`?

**Comment by IndrajeetPatil** [comment]
Thanks to @Yuri05, we now know how to review changes like these appropriately! 

image

File: R/error-checks.R [hunk 1]
```diff
@@ -1,165 +1,168 @@
...
+#' isFileExtension("enum.R", "pkml")
+#' @export
+
+isFileExtension <- function(file, extension) {
+  extension <- c(extension)
+  file_ext <- .fileExtension(file)
+  file_ext %in% extension
+}
+
+#' Check that an a array of values does not include any duplicate
```
Comment (PavelBal) [inline R/error-checks.R:123]: Typo - remove the "a".

**Comment by pchelle** [comment]
> @msevestre The same issue as [here](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/commit/9f1c4f4b99630ff84a48fa38d687aa27d91d8661#commitcomment-65806469) is observed in this PR.
> 
> This is good. Because based on what is common to both of your machines, we can figure out what is causing these massive changes.
> 
> Are you by any chance using `VS Code` instead of `RStudio`?

Using RStudio, but I made a copy/paste of the first PR from Github. Should I use a styler to check it prevents that ?

**Comment by msevestre** [comment]
@pchelle We added a gitattribute file. Hopefully, this will prevent this from happening in the future

**Comment by IndrajeetPatil** [comment]
@pchelle This is actually related to git settings

I'm assuming if you run the following in Windows Command Prompt, you get `true`:

```git
git config --get core.autocrlf
```

We are dealing with this in #40, so you don't need to do anything in your PR.

File: R/error-checks.R [hunk 2]
```diff
@@ -120,7 +120,7 @@ isFileExtension <- function(file, extension) {
   file_ext %in% extension
 }
 
-#' Check that an a array of values does not include any duplicate
+#' Check that an array of values does not include any duplicate
```
Comment (IndrajeetPatil) [inline R/error-checks.R:123]: This change also needs to be reflected in `.Rd` file.

Comment (msevestre) [reply R/error-checks.R:123]: @pchelle Can you update the documentation so that we can close this PR?

Comment (IndrajeetPatil) [reply R/error-checks.R:123]: Let's merge this for now. I can change this in the `main` branch itself.

**Pull request merged**